pos_accounting/src/pool/delta/operator_impls.rs

```diff
@@ -280,62 +279,3 @@ impl<'a> PoSAccountingDelta<'a> {
         Ok(())
     }
 }
-
-impl<'a> PoSAccountingOperatorRead for PoSAccountingDelta<'a> {
```

Comment:
I think it might be premature to delete this. Are you sure? Maybe we'll need it later when we introduce other operations on accounting.

Reply:
Maybe I don't see the idea you put in it. As for me we have `PoSAccountingView` to get data from deltas, `PoSAccountingStorageRead` for storage. `PoSAccountingOperations` (previously PoSAccountingOperatorsWrite) to perform complex operations. But not sure why dedicated `PoSAccountingOperatorRead` is useful

Reply:
I forgot what I was planning back then, this was like more than 3 months ago. I don't mind removing it, I'm just worried we may need it later. You're free to remove it.